client/src/components/ForgotPassword.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import NavBar from "./NavBar";
+import { connect } from "react-redux";
+import { forgotPassword } from "../actions";
+
+class ForgotPassword extends Component {
+  state = {
+    email: ""
+  };
+
+  handleFieldChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    if (this.state.password === this.state.confirmPassword) {
+      this.props.forgotPassword({
+        email: this.state.email
+      });
+    }
+    this.setState({
+      email: ""
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <NavBar />
+        <form className="ForgotPasswordForm" onSubmit={this.handleSubmit}>
```

Comment:
By BEM convention, since "Forgot Password Form" is a specific variant of the "Form" class, it has the Form class, as well as a "modifier" class, like `Form Form--ForgotPassword`. Don't freak about this stuff too much right now - we can experiment a little with styling at this point, then lock in and standardize in Week 3.

Reply:
Yeah, I had just copied the style of the Registration form page and changed the name. I figured we would go through and make changes later.